### PR TITLE
Replace request with node-fetch

### PR DESCRIPTION
**Pull request by msl8r**
### Change description ###
Request is no longer maintained and has vulnerabilities which has broken the pipeline, so using fetch instead.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by timja — COMMENTED** [review]
How about just use `fetch` instead? it's built into node.js now

File: package.json [hunk 1]
```diff
@@ -31,9 +31,8 @@
     "express": "^4.18.1",
     "js-yaml": "^4.1.0",
     "nanoid": "^4.0.0",
-    "pug": "^3.0.2",
-    "request": "2.88.2",
-    "request-promise": "4.2.6"
+    "node-fetch": "2.6.9",
```
Comment (timja) [inline package.json:34]: this should be built into node directly I believe you shouldn't need this on node 16

File: app/index.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const rp = require('request-promise');
+const fetch = require('node-fetch');
```
Comment (timja) [inline app/index.js:1]: should work without I think
```suggestion
```

File: package.json (hunk 1 above)
Comment (msl8r) [reply package.json]: ah ok, got you. I have removed now

File: app/index.js (hunk 2 above)
Comment (msl8r) [reply app/index.js]: done, thanks!

**Review by timja — APPROVED** [review]

**Pull request merged**